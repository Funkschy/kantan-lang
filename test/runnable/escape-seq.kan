import "io";

def main(): void {
    // "\\"
    // "\\0"
    // "\\n"
    // "\\r"
    // "\\t"
    let n = "\0";
    io.printf("\\\\\\\n");
    io.printf("%d\n", *n);
    io.printf("abc\n");
    io.printf("\tt");
    io.printf("\tt\"\n");
}
