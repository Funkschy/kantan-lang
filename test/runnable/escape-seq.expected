\\\
0
abc
	t	t"
